workshop/exercise-0/README.md

```diff
@@ -36,18 +36,9 @@ The Kubernetes command-line tool, kubectl, can be used to deploy and manage appl
 
 1. Install [kubectl](https://kubernetes.io/docs/tasks/tools/install-kubectl/#install-kubectl)
 
-### Install knctl CLI
-The Knative command-line tool, knctl provides a set of commands to interact with a Knative installation. Let's install knctl as well.
+### Install kn CLI
+The Knative command-line tool, kn provides a set of commands to interact with a Knative installation. Let's install kn as well.
 
-1. Install knctl using one of the prebuilt binaries from their [release page](https://github.com/cppforlife/knctl/releases), and run the following commands:
-
-    ```
-    shasum -a 256 ~/Downloads/knctl-*
-    # Compare checksum output to what's included in the release notes
-
-    mv ~/Downloads/knctl-* /usr/local/bin/knctl
-
-    chmod +x /usr/local/bin/knctl
-    ```
+1. Install the kn client by following their [instructions for installation](https://github.com/knative/client/blob/master/docs/README.md#installing-kn). You should be able to download the latest binary executable, and add it to your path, and ensure the file is executable.
```

Comment:
perhaps add: (via `chmod +x kn`)
Just for those newbies among us.

Reply:
👍